### ISSUE TITLE
Async vs Sync Request

### ISSUE THREAD

**Issue by aditya-cherukuri** [irazasyed/laravel-gamp#8]
While trying to send page views hits and e-commerce hit if I send synchronous requests then I can see the data on my Google Analytics dashboard.. When I do the same with async request I don't see any hits. Any idea what the problem could be?

I'm switching the request type using the parameter in the config gamp.php file
**'async_requests' => false,**

**Comment by irazasyed** [comment]
@aditya-cherukuri Sorry for the delay. It seems like i totally missed this ticket. Async request could take few seconds, have you checked that?

Also check your logs for errors. Will have to debug this.

**Comment by aditya-cherukuri** [comment]
@irazasyed I have waited for quite a while on events to show up on the Analytics dashboard. Most of the time I don't see them.

I have checked Laravel logs but I don't see anything related to requests failing. Any other logs I need to check?

I also tried checking the AnalyticsResponse object as suggested in https://github.com/theiconic/php-ga-measurement-protocol#asynchronous-requests-non-blocking and I don't see anything different from the suggested results.
